edgexfoundry/device-sdk-go
# refactor!: Replace internal topics from config with new constants

BREAKING CHANGE: Internal topics no longer configurable, except the base topic.

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-sdk-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  TBD

## Testing Instructions
Run non-secure EdgeX stack
Build and run device simple locally (kill and restart for hybrid mode)
Verify the log has on following start-up :
```
msg="Subscribing to command requests on topic: edgex/device/command/request/device-simple/#"
msg="Responses to command requests will be published on topic: edgex/response/device-simple/<requestId>"
"Subscribing to System Events on topic: edgex/system-events/core-metadata/+/+/device-simple/#
```
The verify the debug messages when publishing are:
```
published to MessageBus on topic: edgex/events/device-simple/Simple-Device/Simple-Device01/Switch
```


## New Dependency Instructions (If applicable)
